### PR TITLE
fix: get the true value of the contributor number(> 5000)

### PR DESCRIPTION
**Pull request by TRebirthC**

### CODE REVIEW

File: packages/integration/controllers/projectContributors.js [hunk 1]
```diff
@@ -75,13 +75,22 @@ async function getProjectContributors(url) {
...
-            contributors =
-              contributorsArrays != undefined && contributorsArrays.length > 0
-                ? contributorsArrays.join('')
-                : '';
+          const repoName = url.match(/\/github\.com\/(.*)/)[1];
+          const content = $(`a[href="/${repoName}/graphs/contributors"]`).text();
+          if (content.length !== 0) {
+            const regex = /(\d{1,3}(,\d{3})*(\.\d+)?)/g;
+            const contributorsArrays = content.match(regex);
+            let contributorsNumber1, contributorsNumber2;
```
Comment (Jiannan-dev) [inline packages/integration/controllers/projectContributors.js]: change name

File: packages/integration/controllers/projectContributors.js [hunk 2]
```diff
@@ -75,13 +75,22 @@ async function getProjectContributors(url) {
...
+            contributorsNumber1 = contributorsArrays[0] ? contributorsArrays[0].replace(/,/g, '') : null;
```
Comment (Jiannan-dev) [inline packages/integration/controllers/projectContributors.js]: contributorsArrays[0]?.replace(/,/g, '')

File: packages/integration/controllers/projectContributors.js [hunk 3]
```diff
@@ -101,17 +110,28 @@ async function getProjectContributors(url) {
...
 
-  const contributorsList = await request.json();
+  let contributorsList;
+  // avoid situations where the project is empty
+  if (request.length > 0) {
+    contributorsList = await request.json();
+  } else {
+    contributorsList = [];
+  }
   return contributorsList;
```
Comment (Jiannan-dev) [inline packages/integration/controllers/projectContributors.js]: return request.length > 0?await request.json():[]

**Review by Jiannan-dev — CHANGES_REQUESTED** [review]

File: packages/integration/controllers/projectContributors.js (hunk 1 above)
Comment (TRebirthC) [reply packages/integration/controllers/projectContributors.js]: done

File: packages/integration/controllers/projectContributors.js (hunk 2 above)
Comment (TRebirthC) [reply packages/integration/controllers/projectContributors.js]: done

File: packages/integration/controllers/projectContributors.js (hunk 3 above)
Comment (TRebirthC) [reply packages/integration/controllers/projectContributors.js]: done

**Review by Jiannan-dev — APPROVED** [review]

**Pull request merged**